src/lib/config/ModelConfig.cc

```diff
@@ -886,6 +886,19 @@ void ModelConfig::postValidation() {
   // Record any unlinked port names
   for (const auto& prt : portnames)
     invalid_ << "\t- " << prt << " has no associated reservation station\n";
+
+  // Ensure LSQ-L1-Interface Load/Store Bandwidth is large enough for vector
+  // unit
```

Comment:
This comment doesn't really make sense to me. The load/store bandwidth needs to be large enough to accomodate a full vector load in a single request request; its to do with the vector length, not the vector unit arrangement

Reply:
I've changed this comment in the latest commit to make this correct and more informative